i18n/en.json

```diff
@@ -1,4 +1,8 @@
 {
+  "system_notice.title": "<strong>System Message</strong><br>from Mattermost",
```

Comment:
Maybe `<br>` is not necessary here?

Reply:
Yeah it's needed on the next line and I didn't want to split it up into two different messages

Reply:
👍 (I just noticed that it's not the same as https://github.com/mattermost/mattermost-webapp/pull/1118/files#diff-08e5c13a7cc7c4fdc5ec0c7847bc7679R24)